l10n_es_ticketbai_api/tests/test_l10n_es_ticketbai_customer_invoice.py

```diff
@@ -0,0 +1,329 @@
+# Copyright 2021 Binovo IT Human Project SL
```

Comment:
Habéis quitado los `commit` en los test `manuales`. El flag `send_to_tax_agency` está deshabilitado por defecto, el desarrollador debe cambiarlo manualmente para ejecutar los test de envío a las Haciendas. El objetivo de los `commit` en estos tests es realizar múltiples envíos cumpliendo el encadenamiento.

Si os parece mejor podéis eliminar todos los test manuales, aquellos que sólo se ejecutan si se activa manualmente el flag `send_to_tax_agency` en el fichero de test `common.py`.

Reply:
Considero que al ser parte de tests el commit no es necesario. Haya o no commit, si algo falla en el test unitario se revertirán los cambios realizados.